cluster-autoscaler/Makefile

```diff
@@ -1,5 +1,6 @@
 all: build
 
+DOCKERFILE?=../builder
```

Comment:
Please rename to BUILDER_DOCKERFILE_DIR.

Reply:
renamed